selfdrive/controls/controlsd.py

```diff
@@ -217,6 +214,15 @@ def set_initial_state(self):
       if any(ps.controlsAllowed for ps in self.sm['pandaStates']):
         self.state = State.enabled
 
+    if SIMULATION:
+      ignore = ['driverCameraState', 'managerState']
+      # block until get the available streams
+      available_streams = VisionIpcClient.available_streams("camerad", block=True)
```

Comment:
does this block until the vipc server is up? controlsd shouldn't hang if camerad doesn't start